### ISSUE TITLE
Bump leaflet dependency to 1.9.1 / 1.9.2?

### ISSUE THREAD

**Issue by ajmas** [vue-leaflet/vue-leaflet#239]
The current release of leaflet is 1.9.2, so thinking we should bump the dependency version?

At the same time, due to what may be a breaking change in 1.9.2, per issue #236, maybe we should change dependency to 1.9.1 until the issue is identified?

**Comment by Vectorial1024** [comment]
I happened to use this library together with leaflet 1.9.1 recently, and have encountered a problem: the map cannot zoom in by mouse wheel, but when I click the "zoom out" button on the leaflet component, all tiles disappeared with this console error:

```
Uncaught Error: Attempted to load an infinite number of tiles
[...]
```

It was very strange, so I cloned the demo project for sanity check. (Demo project: https://github.com/vue-leaflet/vue3-demo-project/blob/master/src/App.vue) It turns out the demo project did not have this problem. When I looked deeper, the demo project and the project I was working on were using different leaflet versions. Demo was using leaflet 1.7.1, and then my side was using leaflet 1.9.1. Downgrading my leaflet to match the demo project fixed the problem on my side.

This is not a problem on this side, but I would not recommend bumping the dependency this early because of this leaflet problem. This problem simply confused me strongly.

------

Perhaps we could try increasing the leaflet version one by one until we find this problem, and then we will know what is the range of leaflet version that we can safely use.

**Comment by Vectorial1024** [comment]
Quick update: turns out I was not alone; someone else also encountered similar problems https://github.com/vue-leaflet/vue-leaflet/issues/238

**Comment by mikeu** [comment]
Leaflet is a peer dependency of vue-leaflet, for exactly this reason. Each project is free to select its own version of Leaflet to install alongside this library. Leaflet has since released v1.9.3, so I would be interested in hearing whether anyone is still encountering issues with it, particularly in conjunction with our new [v0.8.0](https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.8.0) update.

**Comment by rafaellehmkuhl** [comment]
> Leaflet is a peer dependency of vue-leaflet, for exactly this reason. Each project is free to select its own version of Leaflet to install alongside this library. Leaflet has since released v1.9.3, so I would be interested in hearing whether anyone is still encountering issues with it, particularly in conjunction with our new [v0.8.0](https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.8.0) update.

Leaflet v1.9.3 solves the problem of receiving "L is not defined" when adding an LPopup.
